app/views/templates/tests-by-category-widget.html.slim

```diff
@@ -0,0 +1,66 @@
+.panel.panel-default
+  .panel-heading
+    .row
+      .col-md-6
+        h3.panel-title Tests by category
+      .col-md-6
+        div style='float:right;'
+          a.btn.btn-default.btn-xs href='#' role='button'
+            i.glyphicon.glyphicon-filter
+            drop constrain-to-scroll-parent='true' constrain-to-window='true' open-on='"click"' position='"left top"' tether-options='{offset: "-5px -5px"}' before-close='beforeClose()'
```

Comment:
When I tried it, this button would open the dropdown, but not close it when I clicked on it again. It should.